### DISCUSSION TITLE
AIYATO Smart Bulb | Tasmota not working

### DISCUSSION THREAD

**Discussion by dhavaldesai-HA** [arendst/Tasmota#21973] [Q&A]
Hi everyone,

I ordered two AIYATO smart bulbs thinking that they are preflashed with Tasmota. It says on Tasmota blakadder compatibility page as well. When I turn it of, it announced a wifi network with SSID CozyLife_V2N1. I can connect to it as well but that is about it. 192.168.4.1 doesn't take me to tasmota config page. 

Did someone have similar issue in the past? I'd be thankful if someone can help here. 

Thanks,
Dhaval

SSID:	CozyLife_V2N1
Protocol:	Wi-Fi 4 (802.11n)
Security type:	Open
Network band:	2.4 GHz
Network channel:	5
Link speed (Receive/Transmit):	65/65 (Mbps)
Link-local IPv6 address:	fe80::dd3c:22c6:40a9:657f%5
IPv4 address:	192.168.4.2
IPv4 DNS servers:	192.168.4.1
Manufacturer:	Intel Corporation
Description:	Intel(R) Wi-Fi 6 AX200 160MHz
Driver version:	23.60.1.2
Physical address (MAC):	E0-D4-E8-77-56-C3

**Comment by sfromis** [comment]
To me, that SSID suggests that the bulbs do not have Tasmota preflashed, as the name "CozyLife" does seem to be for their own firmware instead.

**Comment by tech1977** [comment]
Hi,
I also have a GU10 AIYATO CozyLife bulb and I would like to install the Tasmota firmware

**Reply by sfromis** [reply]
If you are seeing a SSID (in pairing mode) with "CozyLife", it does not have Tasmota, but it *might* be possible to do wired flashing. However, getting access to the innards of GU10 bulbs is usually difficult without damaging it. And if looking up the MAC you see in pairing mode does not indicate "Espressif", it is not even a compatible chip.
